youtube_dl/downloader/rtmp.py

```diff
@@ -104,7 +104,8 @@ def run_rtmpdump(args):
         live = info_dict.get('rtmp_live', False)
         conn = info_dict.get('rtmp_conn', None)
         protocol = info_dict.get('rtmp_protocol', None)
-
+        no_resume = info_dict.get('no_resume', False)
```

Comment:
Thank you for this hint, I wasn't aware of this. I'm going to add it in the YoutubeDL.py. I needed to create this new property because rtl2 doesn't accept the "--resume" argument which was always appended by default.